tf2_ros/src/tf2_ros/buffer_interface.py

```diff
@@ -45,7 +45,7 @@ def __init__(self):
         self.registration = tf2_ros.TransformRegistration()
 
     # transform, simple api
-    def transform(self, object_stamped, target_frame, timeout=rospy.Duration(0.0), new_type = None):
+    def transform(self, object_stamped, target_frame, timeout = Time(), new_type = None):
```

Comment:
Time and Duration are not the same. This should be Duration.

Reply:
Fixed